Sources/PayPal/PayPalDelegate.swift

```diff
@@ -0,0 +1,25 @@
+import Foundation
+#if canImport(PaymentsCore)
+import PaymentsCore
+#endif
+
+/// PayPal delegate to handle events from PayPalClient
+public protocol PayPalDelegate: AnyObject {
```

Comment:
It looks like we are never using the client piece of the function signatures in the delegate. I think it's probably safe to remove unless there is a specific reason that was included that's not illustrated in this PR. It'll clean up our implementation as well as the merchant implementation of our delegate if it's not needed.

Reply:
Yeah we're mostly following an [iOS pattern](https://www.swiftbysundell.com/articles/delegation-in-swift/#protocols) here similar to UICollectionView, UITableView etc. This is a good way to callout to merchants that it's a delegate method.

Reply:
Gotcha, I think that's fine then, but we should make the first parameter name anonymous to match how other delegates are formatted in the [swift docs](https://docs.swift.org/swift-book/LanguageGuide/Protocols.html) and [UITableView protocol functions](https://developer.apple.com/documentation/uikit/uitableviewdelegate/1614883-tableview) vs it being a named param.